src/index.ts

```diff
@@ -96,6 +227,11 @@ export default class Account {
     })
   }
 
+  /**
+   * Returns a `Boolean` determining if the account is empty.
+   *
+   * @return {boolean} if account is empty
```

Comment:
Could you please add `boolean` instead to the method's annotation? `isEmpty(): boolean`

Reply:
updated!